### ISSUE TITLE
[Nav] Fix A11y issues

### ISSUE THREAD

**Issue by NicoPlyley** [phxlab/phxlab#75]
Improper use of nav element, the links should be a ul

**Comment by NicoPlyley** [comment]
Was already fixed in previous commit
